test/helloworker.test.ts

```diff
@@ -102,6 +103,28 @@ describe('helloworker', () => {
       return Promise.resolve('+1');
     };
 
+    let listKVCalled = false;
+    countryCodeKV.list = (
+      params:
+        | {
+            prefix?: string | undefined;
+            limit?: number | undefined;
+            cursor?: string | undefined;
+          }
+        | undefined,
+    ): Promise<CloudflareWorkerKVList> => {
+      listKVCalled = true;
+      let prefix = 'empty';
+      if (params) {
+        prefix = params.prefix ? params.prefix : 'missing';
```

Comment:
You could use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to shrink this down to:

```suggestion
        prefix = params.prefix ?? 'missing';
```